src/components/ebay-avatar/examples/signedout.marko

```diff
@@ -0,0 +1 @@
+<ebay-avatar a11y-text="Signed out" variant="signedout" ...input/>
```

Comment:
Is this `variant="signedout"` a remnant from an earlier iteration? It looks like you aren't using `input.variant` at all, instead you just check if `input.username` exists.